nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -355,6 +357,10 @@ public String currentHash() {
     return reference.getHash();
   }
 
+  public String initialHash() {
```

Comment:
does this really have to be public? This should rather have a `@VisibleForTesting` annotation and remove the `public`